libnavigation-ui/src/main/java/com/mapbox/navigation/ui/map/BuildingExtrusionLayer.java

```diff
@@ -24,15 +24,18 @@
 import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.fillExtrusionOpacity;
 import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.visibility;
 
+/**
+ * This layer handles the creation and customization of a {@link FillExtrusionLayer}
```

Comment:
What is this layer targeted to be compatible with? I see a lot of hardcoded values like `composite` source or `building` source layer. If the goal is to work only with [`streets-v8`](https://docs.mapbox.com/vector-tiles/reference/mapbox-streets-v8/) tiles, then it would be great to emphasize this in the docs.

Reply:
Added to docs